src/test/unit/pid_unittest.cc

```diff
@@ -135,7 +135,7 @@ void setDefaultTestSettings(void) {
     pidProfile->dterm_lowpass2_hz = 0;
     pidProfile->dterm_notch_hz = 260;
     pidProfile->dterm_notch_cutoff = 160;
-    pidProfile->dterm_filter_type = FILTER_BIQUAD;
+    pidProfile->dterm_filter_type = FILTER_PT1;
```

Comment:
Changing an existing and working test seems to be the wrong approach. Instead we should be adding additional tests for the new filters.

Reply:
Thanks.
The most practical way to test both, in their lowpass application, would be to revert this change, and add a test for a PT1 in the gyro filtering application, for example.  
However I expect that the PT1 is sort of covered by the existing test, since the default gyro filtering would affect the D outcome for the D test.  I'm not sure if that's true though, and it's something I wanted to raise in this PR.
The rationale for changing the test went back to when I initially was removing the `BIQUAD` option entirely, which caused the test to fail because there was no biquad option.
Now the biquad option is there to stay, forever most likely, despite being entirely inappropriate for use as a lowpass filter, I will revert the test back to biquad, and have now done so.